codegen/src/main/java/com/paypal/selion/plugins/CodeGeneratorMojo.java

```diff
@@ -163,7 +163,7 @@ private String pathToFolder(File file) {
      * @see org.apache.maven.plugin.AbstractMojo#execute()
      */
     public void execute() throws MojoExecutionException {
-        // iff baseFolder is null or empty, reset it to the default value
+        // if baseFolder is null or empty, reset it to the default value
```

Comment:
`iff` is shorthand for `if and only if`.. which is what the situation / comment is about.